panel/chat/interface.py

```diff
@@ -207,7 +207,8 @@ def _update_input_width(self):
         """
         Update the input width.
         """
-        self.show_button_name = self.width is None or self.width >= 400
+        if self.show_button_name is None:
+            self.show_button_name = self.width is None or self.width >= 400
```

Comment:
Should this run `on_init` or does it already?

Reply:
It's run under init. I think I tried using `on_init` but there was an issue so I dumped it in init
https://github.com/holoviz/panel/blob/357eb332e31b4b83270dd81d373ee23f96e828c1/panel/chat/interface.py#L185C10-L186C35